### PR TITLE
Replace screenshots in docs

### PR DESCRIPTION
**Pull request by nadijagraca**
## Description
- Replaced all screenshots to reflect navigation panel change from dropdown to accordion.
- Adjusted code examples to remove empty space on the left panel (added filter)
- Adjusted pages.md file to reflect new container structure

## Checklist

- [x] I have not referenced individuals, products or companies in any commits, directly or indirectly
- [x] I have not added data or restricted code in any commits, directly or indirectly
- [ ] I have updated the docstring of any public function/class/model changed
- [ ] I have added the PR number to the change description in the changelog fragment, e.g. `Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))` (if applicable)

## Types of changes

- [x] Docs/refactoring (non-breaking change which improves codebase)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: vizro-core/hatch.toml [hunk 1]
```diff
@@ -23,7 +23,7 @@ dependencies = [
 ]
 
 [envs.default.env-vars]
-DASH_DEBUG = "true"
+DASH_DEBUG = "false"
```
Comment (huong-li-nguyen) [inline vizro-core/hatch.toml:26]: Revert change here

**Review by huong-li-nguyen — COMMENTED** [review]
I've noticed that in the floating images and styling images section we now have two cards in each example? Shall we keep it as we've had before and screenshot only the one example?
You can just adapt the screen size such that it looks nice.

But I would only use the code that is required for the output so to have two examples of the same is redundant I think

Screenshot 2023-09-21 at 14 45 30

File: vizro-core/hatch.toml (hunk 1 above)
Comment (nadijagraca) [reply vizro-core/hatch.toml:26]: Great catch! Reverted.

**Comment by nadijagraca** [comment]
> I've noticed that in the floating images and styling images section we now have two cards in each example? Shall we keep it as we've had before and screenshot only the one example? You can just adapt the screen size such that it looks nice.
> 
> But I would only use the code that is required for the output so to have two examples of the same is redundant I think
> 
> Screenshot 2023-09-21 at 14 45 30

I did it to match the other examples that have two cards, but you are right it is a redundant.

File: vizro-core/docs/pages/user_guides/pages.md [hunk 2]
```diff
@@ -7,14 +7,18 @@ The [`Dashboard`][vizro.models.Dashboard] model accepts the `pages` argument, wh
 
 ## Creating a page
 
-A [`Page`][vizro.models.Page] is split up into three main containers:
+A [`Page`][vizro.models.Page] is split up into four main containers:
 
-1. The **control container**  where you can add your `controls` (see e.g. [Filters](filters.md) or [Parameters](parameters.md)) to interact with the dashboard
-2. The **page header** that contains the page title and the theme toggle switch button
-3. The **component container** where you can add your [Charts/Components](components.md) to visualize your data
+1. The **navigation container** where you can customise your `navigation` (see [Dashboard](dashboard.md) and [Navigation](navigation.md) for more information)
```
Comment (maxschulz-COL) [inline vizro-core/docs/pages/user_guides/pages.md:12]:
```suggestion
1. The **navigation container** where you can customize your `navigation` (see [Dashboard](dashboard.md) and [Navigation](navigation.md) for more information)
```

File: vizro-core/docs/pages/user_guides/pages.md [hunk 3]
```diff
@@ -83,8 +87,10 @@ To create and add a page to your dashboard, do the following steps:
 
 !!! note
 
-    Note that a page selector dropdown is automatically added to your dashboard in the top-left of the control container.
-    This dropdown allows navigating through the different pages.
+    Note that an accordion page selector is automatically added to your dashboard in the top-left of the control container.
+    This accordinon allows navigating through the different pages.
```
Comment (maxschulz-COL) [inline vizro-core/docs/pages/user_guides/pages.md:91]:
```suggestion
    This accordion allows navigating through the different pages.
```

File: vizro-core/docs/pages/user_guides/pages.md [hunk 4]
```diff
@@ -83,8 +87,10 @@ To create and add a page to your dashboard, do the following steps:
...
+
+    Accordion page selector will not be added if your dashboard consists of only one page.
```
Comment (maxschulz-COL) [inline vizro-core/docs/pages/user_guides/pages.md:93]:
```suggestion
    The accordion page selector will not be added if your dashboard consists of only one page.
```

**Review by maxschulz-COL — CHANGES_REQUESTED** [review]
Hero work! Found a few small things, but otherwise I think good to go from my side

**Review by huong-li-nguyen — APPROVED** [review]

**Review by maxschulz-COL — APPROVED** [review]

**Pull request merged**